requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy>=1.7
+pandas>=0.15
+python-dateutil>=2.4.2
+pytz>=2015.4
+six>=1.9.0
```

Comment:
Are the modules on this list actually needed other than `pandas`? I don't see them being used anywhere